keyboards/amj96/amj96.h

```diff
@@ -18,25 +18,28 @@
 
 #include "quantum.h"
 
+// readability
+#define ___ KC_NO
```

Comment:
I think this should be fine, as most people don't go looking in the keyboard files for stuff.

Reply:
I use `___` for `KC_NO` in `keyboard.h` files because I find it easier to spot at a glance than `XXX`, because the switch identifiers are shorter length.

I do stick with the standard usage for `keymap.c` though.